src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 3px 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-top: 40px;
+  }
+
+  &__code {
+    box-sizing: border-box;
+    line-height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-color;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 16px;
```

Comment:
The `box-sizing: border-box;` property is not necessary here because it is inherited from the parent element. It's a good practice to set this property in the universal selector at the beginning of your CSS file to apply it to all elements.